internal/command/scale/machine_defaults.go

```diff
@@ -42,6 +43,10 @@ func newDefaults(appConfig *appconfig.Config, latest api.Release, machines []*ap
 				break
 			}
 		}
+
+		// If we still don't have a guest size, just set it to the default one
+		guest = new(api.MachineGuest)
+		guest.SetSize(deploy.DefaultVMSize)
```

Comment:
Can we do this outside and pass last-mile default `guest` as an argument to `newDefaults`?